### DISCUSSION TITLE
[질문] OSI 7계층 관련

### DISCUSSION THREAD

**Discussion by RunaNam** [woowacourse-study/2022-http-web-basic-for-all-developer#9] [Q&A]
조금 벗어난 것일수도 있는데, 정리하다가 궁금한 내용이 생겨서 질문합니다~

1. OSI 7계층을 나눈 이유는 무엇일까요?
개인적으로 생각해본 이유 :이렇게 계층을 나눈 이유는 통신이 일어는 과정을 단계별로 파악할 수 있기 때문이다. 더불어 특정한 곳에 이상이 생겼을 때 이상이 생긴 단계만 고칠 수 있다는 장점을 갖고 있다.
2. OSI 7계층이 나누어져있는데 왜 TCP/IP 4(5)계층이 따로 있는걸까요?
[관련 블로그 포스팅](https://velog.io/@jwkim/cs-nw-osi-tcp-ip)

**Comment by jurlring** [comment]
1. 저도 루나랑 비슷한 생각인뎅 
통신을 할 때, 공통되는 일을 하는 계층을? 나눠서 추상화했다고 생각해요.
추상화의 장점은 유지보수가 쉽다는게 있을 것 같아요!
단점으로는 계층을 나눔으로 발생하는 중복도 있을 것이라고 생각해요.
2. 처음 알았네용~ 신기하당
